lib/tasks/connectivity.rake

```diff
@@ -0,0 +1,172 @@
+# frozen_string_literal: true
+
+# Checks backend connectivity to the various VA machines. Run with
+# `RAILS_ENV=production bundle exec rake connectivity:all`
+# Also does sanity check to ensure that logs directory is writeable
+
+# Allows running on development machines
+Rails.logger = Logger.new(STDOUT)
+
+# For Rx/SM
+REDIS_CONFIG = Rails.application.config_for(:redis).freeze
+Redis.current = Redis.new(REDIS_CONFIG['redis'])
+
+require 'evss/claims_service'
+require 'facilities/async_client'
+require 'gi/client'
+require 'hca/configuration'
+require 'hca/service'
+require 'mvi/service'
+require 'rx/client'
+require 'sm/client'
+
+namespace :connectivity do
+  desc 'Check connectivity to all backend services'
+  task all: [:db, :edu, :evss, :gi, :hca, :logs, :mvi, :redis, :rx, :sm, :statsd, :vha]
```

Comment:
Sentry will print its own message when this runs if it isn't configured correctly:

```
INFO -- sentry: ** [Raven] Raven 2.1.2 configured not to capture errors.
```